icu4j/main/classes/core/src/com/ibm/icu/text/DateIntervalFormat.java

```diff
@@ -1547,6 +1547,93 @@ private void genFallbackForNotFound(String field, StringBuffer skeleton) {
     }
     */
 
+    private String normalizeHourMetacharacters(String skeleton, ULocale locale) {
+        StringBuilder result = new StringBuilder(skeleton);
+    
+        char hourMetachar = '\0';
+        int metacharStart = 0;
+        int metacharCount = 0;
+        for (int i = 0; i < result.length(); i++) {
+            char c = result.charAt(i);
+            if (c == 'j' || c == 'J' || c == 'C') {
+                if (hourMetachar == '\0') {
```

Comment:
same comment as in C about possible need to ignore literal text here....

Reply:
Again, I don't see anything in here that deals with quoted literal text in skeleton strings, so if this is a bug, it's a preexisting bug (or I'm not looking in the right places).